assets/js/components/Table/Table.jsx

```diff
@@ -43,6 +44,59 @@ const Table = ({ config, data = [] }) => {
   const [filters, setFilters] = useState([]);
   const [currentPage, setCurrentPage] = useState(1);
 
+  const searchParamsEnabled = searchParams && setSearchParams;
+
+  const columnFiltersBindToParams = columns.filter(
+    (c) => c.filter && c.filterFromParams
+  );
+
+  useEffect(() => {
+    if (!searchParamsEnabled) return;
+    const currentParams = searchParams;
+    const filtersBindedToQs = filters.reduce((acc, curr) => {
+      const filterBoundToQs = columnFiltersBindToParams.find(
+        (col) => col.key === curr.key
+      );
+
+      if (!filterBoundToQs) return [...acc];
+
+      return [...acc, { key: curr.key, value: curr.value }];
+    }, []);
+
+    filtersBindedToQs.forEach((f) => {
+      currentParams.delete(f.key);
+
+      f.value.forEach((v) => {
+        currentParams.append(f.key, v);
+      });
+    });
```

Comment:
Can we have a function for this? Something like `updateParameters` would be ok ç_ç

Reply:
done